### PR TITLE
Rebased component templates on vue

### PR DESCRIPTION
**Pull request by roll**
- fixes [partially] #92

---

This PR includes:
- all jinja2 templates converted to vue.js
- basic temporal flask-to-vue routing

It doesn't include:
- tests for created vue components
- rebase of interactive elements to vue

### CODE REVIEW

**Comment by roll** [comment]
@amercader 
please take a look

File: goodtablesio/integrations/github/blueprint.py [hunk 1]
```diff
@@ -89,7 +100,11 @@ def github_settings():
                               GithubRepo.name).
                      all())
 
-    return render_template('github_settings.html', sync=sync, repos=repos)
+    return render_template('index.html', route='GithubSettings', props={
+        'userName': getattr(current_user, 'display_name', None),
+        'sync': sync,
+        'repos': repos,
```
Comment (amercader) [inline goodtablesio/integrations/github/blueprint.py]: This is not JSON serializable so I'm getting an exception on `/github/settings`. Maybe use `[r.to_json() for r in repos]`?

File: goodtablesio/integrations/s3/blueprint.py [hunk 2]
```diff
@@ -29,19 +29,21 @@ def record_params(setup_state):
...
 @s3.route('/settings')
 @login_required
 def settings():
-
     buckets = get_user_buckets(current_user.id)
-
-    return render_template('s3_settings.html', buckets=buckets)
+    return render_template('index.html', route='S3Settings', props={
+        'userName': getattr(current_user, 'display_name', None),
+        'buckets': buckets,
```
Comment (amercader) [inline goodtablesio/integrations/s3/blueprint.py]: Same as in the GitHub blueprint, not JSON serializable

**Comment by roll** [comment]
You right my db seems seriously out of sync after latest changes. I've fixed front test runner so I'll be able to write some tests (also back tests for enpoints html return). 

Hopefully it will be ready to merge on Monday morning because then I'll be away for the second part of the day.

**Comment by roll** [comment]
@amercader 
I've fixed all github problems but I can't add S3 bucket for now (could you send me your env file).

So we could:
- merge this PR now if it's blocking for Sam and I'll be sending next PRs for #92 then
- or I will write js unit tests for it tomorrow if it's not blocking for Sam

**Comment by amercader** [comment]
We can probably merge when the tests are in place. I'll send you the S3 env vars

**Pull request merged**